check for idle docker agents every 30 min

e-carlin opened the pull request:
Fix #2105

robnagler commented on sirepo/job_driver/docker.py:
```diff
@@ -225,6 +230,21 @@ def _init_hosts(cls):
+
+        if not self._idle_timer:
+            self._idle_timer = tornado.ioloop.IOLoop.current().call_later(
+                cfg.idle_check_mins * 60,
```
let's use secs, since everything else in secs (except run time which is in hours)

robnagler commented on sirepo/job_supervisor.py:
```diff
@@ -493,6 +485,16 @@ def _create_op(self, opName, req, **kwargs):
         return o
 
+    def _get_max_run_secs(self, op_name, kind, run_mode):
+        pkdp('{} {} {}', op_name, kind, run_mode)
```
pkdlog

e-carlin replied:
This should've been on my #1435 branch. Reverted.

robnagler commented on sirepo/template/myapp.py:
```diff
@@ -25,6 +26,9 @@
 
 
 def get_data_file(run_dir, model, frame, options=None, **kwargs):
+    if 'suffix' in options and options.suffix == 'sr_long_analysis':
```
options may be None so this will fail.

e-carlin replied:
Thanks, this should be on my #1435 branch which is still WIP. Reverting.

e-carlin commented:
@robnagler I addressed your comments. Two were related to accidental commits (I reverted them and cherry picked them to the correct branch) and the third (secs vs mins) I fixed. Ready for another review.
